content/docs/vmclarity/getting-started/deploy-docker/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Deploy on Docker
+weight: 110
+---
+
+## Prerequisites
+
+* Install [Docker](https://docs.docker.com/get-docker/).
+
+## Deployment steps
+
+To run VMClarity in Docker on a local machine, complete the following steps.
+
+1. Download the latest VMClarity release.
+
+    ```shell
+    wget https://github.com/openclarity/vmclarity/releases/download/v{{< param "latest_version" >}}/docker-compose-v{{< param "latest_version" >}}.tar.gz
+    ```
+
+1. Create a new directory, extract the files and navigate to the directory.
```

Comment:
Shouldn't we properly number the list items like 1., 2., 3., .. :D not sure why it is 1. everywhere it might bother just my OCD

Reply:
I think this is just to simplify maintenance whenever you need to add a step in the middle. If you look into the preview, the numbering is correct https://deploy-preview-23--hugo-openclarity.netlify.app/docs/vmclarity/getting-started/deploy-docker/